src/multivariate/mvlognormal.jl

```diff
@@ -27,14 +27,17 @@ abstract type AbstractMvLogNormal <: ContinuousMultivariateDistribution end
 
 function insupport(::Type{D},x::AbstractVector{T}) where {T<:Real,D<:AbstractMvLogNormal}
     for i=1:length(x)
-      @inbounds 0.0<x[i]<Inf ? continue : (return false)
+        @inbounds 0.0<x[i]<Inf ? continue : (return false)
     end
-    true
+    return true
```

Comment:
This doesn't sound useful to me. Do you follow a particular style guide?

Reply:
both of these recently developed guides follow the convention of always using an explicit `return` keyword with long-form functions
https://github.com/jrevels/YASGuide
https://github.com/invenia/BlueStyle

Reply:
OK. I had another convention in mind, which is that `return` must be used when the control flow is complex, i.e. the function doesn't just have a single return path. Not sure it's written anywhere.

Note that you haven't changed all occurrences (see e.g. `suffstats`). I'm not sure it makes sense to enforce this kind of conventino unless we enable an automated formatter like JuliaFormatter.